### PR TITLE
Issue330

### PR DESCRIPTION
**Pull request by qlli**
When the game starts and analyses the var definition in main.spx, 
the sprite is automatically bound if assets/sprites/{name}/index.json exists,
and the sound is automatically bound if assets/sounds/{name}/index.json exists.
No effect on the sprites and sounds in ZOrder of stage config.
[Archive.zip](https://github.com/user-attachments/files/17069361/Archive.zip)

### CODE REVIEW

File: sprite.go [hunk 1]
```diff
@@ -494,6 +494,7 @@ func (p *Sprite) Die() {
 		p.goAnimate(aniName, ani)
 	}
 
+	p.Stop(OtherScriptsInSprite)
```
Comment (nighca) [inline sprite.go:497]: 需要 rebase 下 goplus/main？

**Comment by JiepengTan** [comment]
52e4164d-22c2-478f-a70d-ba496848f38c

@qlli

**Review by JiepengTan — APPROVED** [review]
Please use rebase instead of merge to synchronize the changes from the remote main branch.

File: game.go [hunk 2]
```diff
@@ -354,6 +358,17 @@ func (p *Game) startLoad(fs spxfs.Dir, cfg *Config) {
 	p.windowHeight_ = cfg.Height
 }
 
+func (p *Game) canBindSprite(sprite Spriter, name string) bool {
```
Comment (JiepengTan) [inline game.go:361]: this param is useless

Comment (qlli) [reply game.go:361]: ok. Done in pr 336.
https://github.com/goplus/spx/pull/336
The pr 335 contains other unexpected commit.
Use pr 336 instead.

**Pull request closed without merging**